### ISSUE TITLE
Mention the version of Gateway API are supported by Linkerd (If not all)

### ISSUE THREAD

**Issue by GTRekter** [linkerd/website#1868]
As a reader, I want to know if Linkerd supports all versions of the Gateway API by default, so I can configure my cluster accordingly.

If it support all write a note about it. If not, then list the supported versions.

**Comment by kflynn** [comment]
Yeah, this is a good shout. Where would you think it most useful to have this information?
